lib/buyers/tests/test_api.py

```diff
@@ -374,3 +374,22 @@ def test_bad_uuid(self):
     def test_empty_post(self):
         res = self.client.post(self.list_url, data={})
         eq_(res.status_code, 422)
+
+
+class TestBuyerClose(APITest):
+
+    def setUp(self):
+        self.uuid = 'sample:uid'
+        self.buyer = Buyer.objects.create(uuid=self.uuid)
+        self.url = reverse('generic:buyer-close', kwargs={'pk': self.buyer.pk})
+
+    def test_close(self):
+        res = self.client.post(self.url, data={'uuid': self.uuid})
+        eq_(res.status_code, 204)
+        eq_(self.buyer.reget().active, False, res.content)
```

Comment:
I think it's worth testing that the email is blank and that the uuid is different